docs/integrations/wallets/ledger/ledger-native.md

```diff
@@ -80,8 +82,12 @@ Ledger NanoS users should install the Astar XL version of the app, shown in the
 
 - Open a browser and visit the [Astar Portal](https://portal.astar.network).
 - Connect the Polkadot.js extension to the Portal.
-- Select the Ledger account that was imported during the last step. It can now be used to participate in dApp staking.
+- Select the Ledger account that was imported during the last step. And check the toggle so Portal knows the device is a ledger. It can now be used to participate in dApp staking.
```

Comment:
```suggestion
- Select the Ledger account that was imported during the last step.  Check the toggle so Portal knows the device is a ledger. It can now be used to participate in dApp staking.
```